eslint/rules/no-sync-mongo-methods.ts

```diff
@@ -1,5 +1,5 @@
 import { ESLintUtils } from '@typescript-eslint/utils';
```

Comment:
One note: because `Model`s wrap `Mongo.Collection` instead of inheriting from it, this will not catch synchronous reads against `Model`s.

I'm choosing to defer this for the time being because I want to rework this check a little bit. (Specifically: I want to broaden it to more than just Mongo methods, now that Meteor is increasingly supporting async versions of core APIs). Given the current state of our code and infra, I'm reasonably confident that it'll be easy to go through and clean up any accidental synchronous calls introduced between now and then.